plugins/slack_app/README.md

```diff
@@ -0,0 +1,12 @@
+# Slack App Plugin
+
+This plugin adds a `/deploy` command to Slack which creates Samson deployments.
+To activate:
+
+1. Go over to Slack and create a [new app](https://api.slack.com/apps/new). Many of the fields are completely up to you, but these are important:
+  - Grab the Client ID and secret and verification token, and record them in your environment as `SLACK_CLIENT_ID`, `SLACK_CLIENT_SECRET`, and `SLACK_VERIFICATION_TOKEN`.
+  - Add `https://your.samson.tld/slack_app/oauth` as a redirect URI.
+  - Enable interactive messages, and set the URI to `https://your.samson.tld/slack_app/interact`.
+  - Add a `/deploy` slash-command, and set its URL to `https://your.samson.tld/slack_app/command`.
+2. Visit https://your.samson.tld/slack_app/oauth, and click the "Connect to Slack" button.
+3. Switch back over to Slack, and try out the `/deploy` command!
```

Comment:
Hi @ben, there is an issue when I work by this README:
if I try the `/deploy` command there is an exception:
```bash
2016-08-07T08:01:52.804249+00:00 app[web.1]: Completed 500 Internal Server Error in 15ms (ActiveRecord: 4.5ms)
2016-08-07T08:01:52.809853+00:00 app[web.1]: 
2016-08-07T08:01:52.809855+00:00 app[web.1]: NoMethodError (undefined method `captures' for nil:NilClass):
2016-08-07T08:01:52.809856+00:00 app[web.1]:   plugins/slack_app/app/controllers/slack_app_controller.rb:55:in `command_response'
2016-08-07T08:01:52.809857+00:00 app[web.1]:   plugins/slack_app/app/controllers/slack_app_controller.rb:34:in `command'
```
I didn't find any example on how to use the `/deploy` command.